### PR TITLE
= kamon-scala: cross build for Scala 2.12

### PR DESCRIPTION
**Pull request by cory-p-oncota**

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -24,32 +24,29 @@ import scalariform.formatter.preferences._
...
     ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) },
-    scalaVersion                    := SVersion,
-    crossScalaVersions              := Seq("2.10.5", SVersion),
-    resolvers                       ++= Dependencies.resolutionRepos,
-    fork in run                     := true,
-    parallelExecution in Global     := false,
-    testGrouping in Test            := singleTestPerJvm((definedTests in Test).value, (javaOptions in Test).value),
-    javacOptions                    := Seq(
+    scalaVersion                        := SVersion,
+    crossScalaVersions                  := Seq("2.10.5", "2.11.8", "2.12.1"),
```
Comment (agarbutt) [inline project/Settings.scala]: You might want to leverage the SVersion value defined above.

File: project/Settings.scala [hunk 2]
```diff
@@ -24,32 +24,29 @@ import scalariform.formatter.preferences._
...
+    resolvers                           ++= Dependencies.resolutionRepos,
+    fork in run                         := true,
+    parallelExecution in Global         := false,
+    testGrouping in Test                := singleTestPerJvm((definedTests in Test).value, (javaOptions in Test).value),
+    javacOptions                        := Seq(
       "-Xlint:-options",
       "-source", JavaVersion, "-target", JavaVersion),
```
Comment (agarbutt) [inline project/Settings.scala:40]: Not sure that bumping the javac -target to 1.8 would support older versions of the JVM, Akka, and Scala 2.10.

Comment (cory-p-oncota) [reply project/Settings.scala:40]: I believe you are correct. I misconstrued the warnings from scalac; javac options can remain as 1.6.

**Comment by dpsoft** [comment]
@cory-p-oncota @agarbutt thanks again guys!!!

**Pull request merged**